examples/visualization/plot_clickable_image.py

```diff
@@ -24,10 +25,12 @@
 
 # Set parameters and paths
 plt.rcParams['image.cmap'] = 'gray'
-im_path = op.join(op.dirname(mne.__file__), 'data', 'image', 'mni_brain.gif')
+
+im_path = op.join(op.dirname(__file__), '..', '..', 'mne', 'data', 'image',
+                  'mni_brain.gif')
 
 # We've already clicked and exported
-layout_path = op.join(op.dirname(mne.__file__), 'data', 'image')
+layout_path = op.join(op.dirname(__file__), '..', '..', 'mne', 'data', 'image')
```

Comment:
Good call, it is indeed missing

https://github.com/mne-tools/mne-python/blob/master/setup.py#L106